lib/active_model/serializer/include_tree.rb

```diff
@@ -0,0 +1,77 @@
+module ActiveModel
+  class Serializer
+    # Description
+
+    class IncludeTree
+      class << self
+        # Builds an IncludeTree from a comma separated list of dot separated paths (JSON API format).
+        # @example `'posts.author, posts.comments.upvotes, posts.comments.author'`
+        #
+        # @param [String] included
+        # @return [IncludeTree]
+        #
```

Comment:
Also, some of these styles will need to be autocorrected if we merge https://github.com/rails-api/active_model_serializers/pull/1183